app/views/course/achievements/_form.html.slim

```diff
@@ -7,29 +7,7 @@
 
   = f.input :draft
 
-  - if @achievement.try(:persisted?)
```

Comment:
because this is the form for achievement. If this form is for creating a new achievement, we will not show the add/edit condition view.